lib/mte/mteBase.pas

```diff
@@ -200,6 +200,7 @@ class function TPluginHelpers.CreateNewBasePlugin(var list: TList; filename: str
   plugin: TBasePlugin;
 begin
   Result := nil;
+  LoadOrder := PluginsList.Count + 1;
```

Comment:
I'm pretty sure `PluginsList.Count` should already be at the correct ordinal because `LoadOrder` is 0-based.

Reply:
I just restored that line from before my changes, the `+ 1` was there originally.

Reply:
huh, ok.  might have been a bug then.  I'd have to look closer to see if that is indeed the case or if something else is going on.  it's fine for now I suppose :+1: